src/main/kotlin/org/rust/cargo/runconfig/wasmpack/WasmPackCommandConfigurationType.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.runconfig.wasmpack
+
+import com.intellij.execution.configurations.ConfigurationFactory
+import com.intellij.execution.configurations.ConfigurationTypeBase
+import com.intellij.execution.configurations.ConfigurationTypeUtil
+import com.intellij.execution.configurations.RunConfiguration
+import com.intellij.openapi.project.Project
+import com.intellij.util.PlatformUtils
+import org.rust.ide.icons.RsIcons
+
+class WasmPackCommandConfigurationType : ConfigurationTypeBase(
+    "WasmPackCommandRunConfiguration",
+    "Wasm-pack Command",
```

Comment:
Not sure about this name, especially about capitalization. Probably `wasm-pack command` looks a bit better, but we already have `Cargo Command`. Or maybe even just `wasm-pack`. @Undin what do you think?

Reply:
Discussed in person that `wasm-pack` is better and `Cargo Command` can be renamed to `Cargo` later

Reply:
Should we now use `wasm-pack` naming without `Command` in `WasmPackCommandConfigurationType.ID` too?

Reply:
Yes